README.md

```diff
@@ -274,6 +277,29 @@ out the necessary python commands to run to deactivate your env and remove the c
 
 ```shell script
 make teardown
+### Setup: Nix
+
+NOTE: Nix is an experiment. If you are setting things up with Nix you
+do not need to follow the instructions above about homebrew,
+pre-commit, or pyenv
+
+NOTE: Nix as an experiment means you ask for help in the `#code-nix`
+slack channel. It's not an officially supported development environment.
+
+1. First read the overview in the [Truss Engineering Playbook](https://github.com/trussworks/Engineering-Playbook/tree/main/developing/nix).
+1. Follow the [macOS installation instructions](https://nixos.org/manual/nix/stable/#sect-macos-installation).
+1. Ensure you have `direnv` and a modern `bash` installed. To install
+   globally with nix, run `nix-env -i direnv bash`
+1. Ensure you have run `direnv allow` to set up the appropriate nix
+   environment variables.
+1. Make sure you have disabled any `nodeenv`, `asdf` or any other
+   version switchers.
+1. Run `./nix/update.sh`
```

Comment:
What should I do after running this command? I ran it but don't have `pytest` which doesn't appear in the `default.nix` file. I tried `make venv` and `make setup` but both failed due to missing `pyenv`. Would be nice to include the rest of the steps to get the tests running if possible.

Reply:
I added docs for installing the pip dependencies